pkg/conf/schedulerconf_test.go

```diff
@@ -37,3 +39,34 @@ func TestDefaultValues(t *testing.T) {
 	assert.Equal(t, conf.KubeBurst, DefaultKubeBurst)
 	assert.Equal(t, conf.Predicates, "")
 }
+
+// run some concurrent write/read
+// if the Get and Set function is not thread safe,
+// this will give data race warnings when running with -race flag
+func TestConcurrency(t *testing.T) {
+	wg := sync.WaitGroup{}
+
+	wg.Add(1)
+	go func() {
+		for i := 0; i < 10; i++ {
```

Comment:
Can you move the `10` to a constant and use it here and above in the other goroutine?